### PR TITLE
SDL: Show the actually used sample rate in system info.

### PR DESCRIPTION
**Pull request by hrydgard**
Right now it'll always be the requested 44100 though anyway, since we don't accept other sample rates. That will be changed later probably.

(I'm debugging some strange unnecessary audio underruns on my mac).

### CODE REVIEW

File: SDL/SDLMain.cpp [hunk 1]
```diff
@@ -106,24 +108,25 @@ static void InitSDLAudioDevice(const std::string &name = "") {
...
 	if (!startDevice.empty()) {
-		audioDev = SDL_OpenAudioDevice(startDevice.c_str(), 0, &fmt, &ret_fmt, SDL_AUDIO_ALLOW_FREQUENCY_CHANGE);
+		audioDev = SDL_OpenAudioDevice(startDevice.c_str(), 0, &fmt, &g_retFmt, SDL_AUDIO_ALLOW_FREQUENCY_CHANGE);
 		if (audioDev <= 0) {
 			WLOG("Failed to open audio device: %s", startDevice.c_str());
 		}
 	}
 	if (audioDev <= 0) {
-		audioDev = SDL_OpenAudioDevice(nullptr, 0, &fmt, &ret_fmt, SDL_AUDIO_ALLOW_FREQUENCY_CHANGE);
+		ILOG("SDL: Trying a different sample rate");
```
Comment (unknownbrackets) [inline SDL/SDLMain.cpp:117]: This is actually trying a difference device, they both allow a freq change.

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

File: SDL/SDLMain.cpp (hunk 1 above)
Comment (hrydgard) [reply SDL/SDLMain.cpp:117]: Oh right, was reading carelessly, apparently.

**Comment by hrydgard** [comment]
Added another piece of debug info and made it easier to display more stats, too.

(Having strange issues where it tries to use a slightly higher sample rate than it should on Mac (around 44200Hz), which results in an underrun per second or so. Will log/display stuff until I see the issue...).

**Comment by unknownbrackets** [comment]
There's already an issue reported about that, actually.  See #12705.

-[Unknown]

**Comment by hrydgard** [comment]
Ah, right, that looks like the same thing I'm experiencing. Somehow forgot about that one.

**Pull request merged**